### ISSUE TITLE
lint.xml missing?

### ISSUE THREAD

**Issue by miduch** [xmartlabs/gong#274]
shouldn't lint.xml be included in repo? when building warning can be seen that lint.xml not found

**Comment by mirland** [comment]
Yes, I think it should be tracked. However, right now Gong has not a special linter rule, so the lint.xml will be "empty". However, I will add it to remove the warning!
